packages/shared/lib/services/sync/run.service.ts

```diff
@@ -401,25 +401,17 @@ export default class SyncRun {
     async finishSync(models: string[], syncStartDate: Date, version: string, totalRunTime: number, trackDeletes?: boolean): Promise<void> {
         let i = 0;
         for (const model of models) {
+            let deletedKeys: string[] = [];
             if (!this.isWebhook && trackDeletes) {
```

Comment:
@khaliqgant I have kept the condition here but I am not sure why `!this.isWebhook` is necessary

Reply:
Because a webhook and a sync with track deletes share the same config